src/api/apps/articles/model/save.coffee

```diff
@@ -137,7 +138,17 @@ removeStopWords = (title) ->
             .set('X-Xapp-Token': token)
             .end callback
   async.parallel callbacks, (err, results) =>
-    return cb(err) if err
+    if err
+      console.log chalk.red('[ERROR: articles/model/save @ generateKeywords]'), err.response.error
```

Comment:
Another problem around this, which is the reason it took so long, was because of all the callbacks, errors were getting consumed and passed all around the stack -- and when the error finally landed it contained no valuable info. `error.response.error` fills in details, like req url, error, and stack trace, making debugging much easier.